appveyor.yml

```diff
@@ -34,19 +50,25 @@ pull_requests:
 
 install:
   - cmd: git submodule update --init --recursive
+  - sh: git submodule update --init --recursive
 
 after_build:
   - cmd: nuget pack src\OSPSuite.SimModelSolver_CVODES\OSPSuite.SimModelSolver_CVODES.nuspec -version %app_version%
+  - sh: nuget pack src/OSPSuite.SimModelSolver_CVODES/OSPSuite.SimModelSolver_CVODES_Ubuntu18.nuspec -version $app_version
   - ps: Get-ChildItem .\OSPSuite.SimModelSolver_CVODES.*.nupkg | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
-
+  
 test:
+  # currently tests are run on Windows only
```

Comment:
all the solver functionality is tested as part of SimModel tests anyway; moving from C++/CLI to .Net core/Pinvoke just to run the tests under Linux is too much effort at the moment